trulens_eval/examples/experimental/virtual_example.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # dictionry to TruVirtual (later). This may involve an index of components or
</pre><p>typo: dictionary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Fixed.